cli/resource-mover-cli/src/main/kotlin/net/thunderbird/cli/resource/mover/ResourceMoverCli.kt

```diff
@@ -0,0 +1,32 @@
+package net.thunderbird.cli.resource.mover
+
+import com.github.ajalt.clikt.core.CliktCommand
+import com.github.ajalt.clikt.parameters.options.option
+import com.github.ajalt.clikt.parameters.options.required
+
+class ResourceMoverCli(
+    private val stringResourceMover: StringResourceMover = StringResourceMover(),
+) : CliktCommand(
+    name = "resource-mover",
+    help = "Move string resources from one file to another",
+) {
+    private val from: String by option(
+        help = "Source module path",
+    ).required()
```

Comment:
clikt could check most of the preconditions.

```suggestion
    private val from: File by option(help = "Source module path")
        .file(mustExist = true, canBeFile = false)
        .required()
```

Reply:
Thanks for deep diving into clikt's options, but I don't see a benefit doing so.